documentation/src/test/java/org/hibernate/userguide/envers/EntityTypeChangeAuditTest.java

```diff
@@ -110,11 +110,11 @@ public void test() {
                 assertEquals(
                     "org.hibernate.userguide.envers.EntityTypeChangeAuditTest$ApplicationCustomer",
                     AuditReaderFactory
-                    .get( entityManager )
-                    .getCrossTypeRevisionChangesReader()
-                    .findEntityTypes( 2 )
-                    .iterator().next()
-                    .getFirst()
+                        .get( entityManager )
+                        .getCrossTypeRevisionChangesReader()
+                        .findEntityTypes( 2 )
+                        .iterator().next()
+                        .getFirst()
```

Comment:
Your indentation used here seems inconsistent with that in all the other files modified in this commit.

Reply:
yeah, there are many other inconsistency issue. One question, should we replace 'tab' with empty chars to ensure predictable and consistent code snippet presenting. Currently I only ensure the indentation is correct on web, but mixing of tab and space chars seems not robust.

Reply:
I decided to close this PR temporarily. Especially I think I wanna exclude testing code format change from the scope of this PR (which has been big enough).